src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -4,45 +4,51 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/** User class for SFTP project. Used to gather information about a connection */
+/** Gather information about a connection initiated by the user with the User class */
```

Comment:
```suggestion
/** Gathers information about a connection initiated by the user with the User class */
```
I noticed you removed these, but I believe it is actually standard to use plural verbs to describe the class, method, or field (leaving the _s_ on the end) in your documentation.

I found a style guide that explains this a bit better than I can: https://developers.google.com/style/reference-verbs